maintainers.md

```diff
@@ -0,0 +1,64 @@
+# Maintainers
+
+## Role and Responsibilities
+
+The main duties of lesson maintenance include 
+
+* reviewing and merging new pull requests
+* opening and tracking issues
+* guiding the direction of the lesson
+* keeping the lesson up to date with the standard template
+* participating in discussions of lesson content/formatting on the [maintainers mailing list][maintainers]
+* maintain a welcoming and positive atmosphere for contribution by 
+	* responding promptly to new pull requests or issues
+	* following Software Carpentry's [Code of Conduct][CoC]
+
+The lesson maintainer is not required 
+to submit their own changes to the lesson in response to particular issues, 
+but they are welcome to submit PRs as they choose, like any other community 
+member.  
+
+## New Maintainer Selection Process
+
+* Outgoing maintainer emails [discuss][discuss] with announcement of opportunity: 
+	* Application information: name, github username, statement of intent
+	* Deadline for applications, projected timeline for selection
+	* Name/email of contact for application process (typically outgoing maintainer)
+* Collect applications over stated time period.  
+* Outgoing maintainer and their co-maintainer review applications and choose new maintainer. 
+* Inform new maintainer, and other applicants via email. 
+* Onboard new maintainer
+	* Add new maintainer to [maintainers][maintainers] if not already a member.
+	* Email [maintainers][maintainers] list announcing the change. 
+	* Request push/merge access for new maintainer from Greg or Jonah.  
```

Comment:
Institutionalize with “Greg or Jonah” → “the Executive Director”?  I don't know if we have a better name for the swcarpentry admin hat.